src/index.ts

```diff
@@ -21,6 +21,7 @@ import { TSESLint } from '@typescript-eslint/experimental-utils';
 
 const sonarjsRules: [string, TSESLint.Linter.RuleLevel][] = [
   ['cognitive-complexity', 'error'],
+  ['elseif-without-else', 'error'],
```

Comment:
I believe we should not activate this rule by default (though it seems to be a first rule in this plugin of this kind)

Reply:
Should we document that in the README ?

Reply:
Good idea!

Reply:
@vilchik-elena Could you elaborate on why you think this rule should not be enabled by default?

Reply:
This is a defensive programming rule, not everyone prefers this style programming. It might imply writing an empty else block with the need to explain why it's empty.